osgpkitools/incommon_request.py

```diff
@@ -219,10 +220,12 @@ def submit_request(config, restclient, hostname, cert_csr, sans=None):
             logger.debug('response text: ' + str(response_text))
             response_data = json.loads(response_text)
             response_data = response_data['sslId']
-        else:
+        elif response.status == 401:
             raise AuthenticationFailureException(response.status, "Connection failure to InCommon API. Check your authentication credentials.")
+        else:
+            raise httplib.HTTPException(response.status, "Connection failure to InCommon API")
```

Comment:
Isn't the status a number, can that be the first parameter to an exception?  I think it should be all one string, and include the response.reason like in test_incommon_connection.